frontend/packages/console-app/locales/en/console-app.json

```
@@ -178,6 +178,15 @@
   "Namespace where the API configuration secret is located": "Namespace where the API configuration secret is located",
   "Scheduling disabled": "Scheduling disabled",
   "Approval required": "Approval required",
+  "Display Name": "Display Name",
+  "Status": "Status",
+  "The state of the plugin.": "The state of the plugin.",
+  "Enabled": "Enabled",
+  "A boolean to determine whether or not the plugin renders in console.": "A boolean to determine whether or not the plugin renders in console.",
+  "CSP Violations": "CSP Violations",
+  "Plugin might have violated the Console Content Security Policy.": "Plugin might have violated the Console Content Security Policy.",
```

Should this one be similar to the one on line 235? or can we make it a complete sentence/

```suggestion
  "This plugin might have violated the Console Content Security Policy.": "This plugin might have violated the Console Content Security Policy.",
```

---

Sorry, @opayne1.  I rewrote this while making some additions and just noticed your comment.  The text now reads, `Whether or not the plugin might have violated the Console Content Security Policy.`. WDYT?